docker-compose.yml

```diff
@@ -11,6 +11,34 @@ services:
     volumes:
       - "./docker/sirius/openapi.yml:/tmp/openapi.yml"
 
+  experian-crosscore-mock:
+    image: stoplight/prism:5.11.1
+    command: mock -h 0.0.0.0 -p 4010 -d /tmp/openapi.yml
+    restart: unless-stopped
+    healthcheck:
+      test: wget -O /dev/null -S 'http://0.0.0.0:4010/api/v1/healthcheck' 2>&1 | grep 'HTTP/1.1 200 OK' || exit 1
+      interval: 15s
+      timeout: 10s
+      retries: 3
+    volumes:
+      - "./docker/experian-crosscore/openapi.yml:/tmp/openapi.yml"
+    ports:
+     - 8082:4010
```

Comment:
Preferable not to expose ports that aren't necessary, since you end up competing with other things running on your computer.